releasenotes/notes/py38_end_of_support-fa1fdea6ea02b502.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Removed support for using Qiskit Machine Learning with Python 3.8, reflecting
```

Comment:
I approved this already, and noticed the future tense aspect here for the EOL. Bbut we can of course edit the text later when doing the release as this talks about upcoming EOL which I guess is not far off now given that we are already into October so depending on how the release pans out that. Or we could just simply remove the word "upcoming" from the text and I think it would still be good and would read fine for anyone reading the notes later too.

Reply:
Thanks for noticing. I updated it now to avoid forgetting.